code/modules/projectiles/guns/projectile/constructable/blinder.dm

```diff
@@ -26,14 +27,21 @@
 		flashbulb = null
 	..()
 
-/obj/item/device/blinder/New()
+/obj/item/device/blinder/New(var/empty = FALSE)
 	..()
+	if(empty == TRUE)
+		start_with_bulb = FALSE
 	if(start_with_bulb)
 		flashbulb = new(src)
 	update_verbs()
+	base_desc = desc
+	update_desc()
 
-/obj/item/device/blinder/empty
-	start_with_bulb = FALSE
+/obj/item/device/blinder/proc/update_desc()
```

Comment:
Is there a reason you're not overriding `examine()` instead?

Reply:
I'd have to poke around in examine so I could get the text to be on the same line like it is currently. But yeah, let me try that.